python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -119,6 +122,20 @@ class GrapheneBulkActionStatus(graphene.Enum):
     class Meta:
         name = "BulkActionStatus"
 
+    def to_dagster_run_status(self) -> GrapheneRunStatus:
+        """Placeholder for this PR. Will do a more thurough pass to accurately convert backfill status
+        to DagsterRunStatus in a stacked branch.
+        """
```

Comment:
probably want to edit comment if this is going to land and not be just PR time placeholder